test/spec/support/debug_helper.rb

```diff
@@ -0,0 +1,11 @@
+module DebugHelper
+  def debug?
+    ENV.has_key?('DEBUG_SPECS')
+  end
+
+  def debug(msg = nil, &block)
+    msg = msg || yield
```

Comment:
`puts(block_given? ? yield : msg) if debug?` would keep the intent (not yielding when not debugging) of the logger-alike message block.